bscript/script.go

```diff
@@ -25,6 +25,10 @@ const (
 	ScriptTypeNonStandard = "nonstandard"
 	ScriptTypeMultiSig    = "multisig"
 	ScriptTypeNullData    = "nulldata"
+
+	MaxOps                = 201
```

Comment:
hmm, do we need these?

Reply:
I'll play about with removing them and let you know